app/scripts/modules/amazon/src/loadBalancer/loadBalancer.transformer.ts

```diff
@@ -368,6 +368,7 @@ export class AwsLoadBalancerTransformer {
               stickinessEnabled: targetGroup.attributes['stickiness.enabled'] === 'true',
               stickinessType: targetGroup.attributes['stickiness.type'],
               stickinessDuration: Number(targetGroup.attributes['stickiness.lb_cookie.duration_seconds']),
+              multiValueHeadersEnabled: targetGroup.attributes['multiValueHeaders.enabled'] === 'true',
```

Comment:
should this be `'multiValueHeadersEnabled'`?

Reply:
apologies!
it should be lambda.multiValueHeaders.enabled
that is the name of the flag used by AWS Lambda
I have fixed it and pushed the change
thank you so much for pointing it out
that was a great catch!